### PR TITLE
More Forum Suggestions

### PR DESCRIPTION
**Pull request by vtange**
https://forum.treeofsavior.com/t/translation-dialogue-icbt-2/125056/6

probably the end of that thread

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -3188,7 +3188,7 @@ QUEST_LV_0200_20150323_003187	$Okay.. I think I can eat those Young Leaves.{nl}T
 QUEST_LV_0200_20150323_003188	$My hunger doesn't go away even after eating a lot.{nl}It's actually getting worse, I might as well just die.
 QUEST_LV_0200_20150323_003189	$There must be something to eat around here..{nl}Please.. get me something to eat.{nl}
 QUEST_LV_0200_20150323_003190	$I am so hungry yet something is echoing inside my head..{nl}The sound of a music box..  I can't eat an echo..
-QUEST_LV_0200_20150323_003191	$Ahah, my Liliya..{nl}I probably can never see you again.
+QUEST_LV_0200_20150323_003191	$Ahh, my Liliya..{nl}I will never be able to see you again..
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:3191]: ellipsis (2x)

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -9166,11 +9166,11 @@ QUEST_LV_0200_20151001_009169	$I heard the story about the Revelator who stopped
 QUEST_LV_0200_20151001_009170	I've been to many places before in my life, but this place is really horrible.{nl}Even if I want to rescue the ones who are cursed, the only thing I can do is to help them fall asleep comfortably.
 QUEST_LV_0200_20151001_009171	$I heard a Revelator defeated the Demon Lord attacking the Mage Tower.{nl}That's the most positive news I've heard lately.{nl}All the other news I've heard were horrible.
 QUEST_LV_0200_20151001_009172	$Is it true that some Revelator defeated the Demon Lord who casted the curse in this region?{nl}If that's true, it is so fortunate since we don't have to see those horrible things again.
-QUEST_LV_0200_20151001_009173	$Oh my g... What was I doing?{nl}Ahhh. My Liliya...{nl}
+QUEST_LV_0200_20151001_009173	$Oh godde... What was I doing?{nl}Ahhh. My Liliya...{nl}
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:9169]: Oh godde... -> Oh no...

File: ETC.tsv [hunk 3]
```diff
@@ -9802,7 +9802,7 @@ ETC_20150323_009803	{memo X}Use multiple skills to suppress overkill.
 ETC_20150323_009804	{memo X}Use Status Points obtained through leveling up {nl}    and upgrade your character's abilities.
 ETC_20150323_009805	Elementalist
 ETC_20150323_009806	Vinas Camp
-ETC_20150323_009807	Rishkio City
+ETC_20150323_009807	Rieszhkio City
```
Comment (Sunaries) [inline ETC.tsv]: I asked Xenobius about it. He said:

"Ryškus City, then if you want it to be a "Bright City"
And "sz" is a double consonant used in Polish/Czech languages"

"- At any rate, I would go for "Ryškus" City (masculine) or Ryški City (feminine), definitely not "Rieszhkio""

(without the accents then of course)

File: ETC.tsv [hunk 4]
```diff
@@ -9802,7 +9802,7 @@ ETC_20150323_009803	{memo X}Use multiple skills to suppress overkill.
 ETC_20150323_009804	{memo X}Use Status Points obtained through leveling up {nl}    and upgrade your character's abilities.
 ETC_20150323_009805	Elementalist
 ETC_20150323_009806	Vinas Camp
-ETC_20150323_009807	Rishkio City
+ETC_20150323_009807	Ryškus City
```
Comment (Sunaries) [inline ETC.tsv]: No accents. (is this also covered in the other files where this particular map is mentioned?)

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**